demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -816,3 +823,43 @@ def test_get_unfiltered_changed_files_from_git_in_external_pr_use_case(
 
     if Path.exists(temp_file):
         Path.unlink(temp_file)
+
+
+def test_check_metadata_version_bump_on_content_changes(mocker, repo):
+    """
+    Given: pack with newly added integration.
+    When: Initializing ValidateManager using git.
+    Then: Ensure PackMetadataVersionShouldBeRaisedValidator is initialized and the external args are properly passed.
+    """
+    pack = create_pack_object(["currentVersion"], ["1.1.1"])
+    integration = create_integration_object()
+    pack.content_items.integration.extend(integration)
+    validation_results = ResultWriter()
+    config_reader = ConfigReader(explicitly_selected=["PA114"])
+    mocker.patch.object(
+        Initializer,
+        "get_files_using_git",
+        return_value=({BaseContent.from_path(Path(integration.path)), pack}, {}, {}),
+    )
+    mocker.patch.object(
+        BaseContent,
+        "from_path",
+        return_value=BaseContent.from_path(Path(pack.path), metadata_only=True),
+    )
+    initializer = Initializer(
+        prev_ver="some_prev_ver", execution_mode=ExecutionMode.USE_GIT
+    )
+
+    validate_manager = ValidateManager(
+        validation_results=validation_results,
+        config_reader=config_reader,
+        initializer=initializer,
+    )
+
+    version_bump_validator = None
+    for validator in validate_manager.validators:
+        if isinstance(validator, PackMetadataVersionShouldBeRaisedValidator):
+            version_bump_validator = validator
```

Comment:
why do we need this if we are using the explicitly_selected?

Reply:
This is to check that even though a pack_metadata was not changed, the validation is collected and will run.